@xen-orchestra/xva/templates/vm.mjs

```diff
@@ -86,8 +87,8 @@ export const DEFAULT_VM = {
     snapshot_time: new Date(0),
     snapshots: [],
     start_delay: 0,
-    // suspend_VDI:'OpaqueRef:NULL',
-    // suspend_SR:'OpaqueRef:NULL',
+    suspend_VDI: 'OpaqueRef:NULL',
+    suspend_SR: 'OpaqueRef:NULL',
```

Comment:
These changes are not explained by the commit message, I don't understand why they are necessary.

Reply:
the second commit message say `Fix(vxva): 'INTERNAL_ERROR: [ (Failure "Field suspend_VDI not present  in rpc") ] `
I updated the PR message to reflect this